### PR TITLE
Remove "postedBy" reference

### PR DESCRIPTION
**Pull request by gadogttas**
This isn't implemented until the next section on authentication.

### CODE REVIEW

**Comment by kaqqao** [comment]
It's not enough to remove the reference from chapter 4... Instead, the saving/loading of `postedBy` should be moved to chapter 5. Not sure if/how I can add commits to this same PR, so I'll create another one.

File: content/backend/graphql-java/4-connectors.md [hunk 1]
```diff
@@ -137,7 +136,6 @@ For this project, you'll use MongoDB as the persistent storage, but by following
 	                doc.get("_id").toString(),
 	                doc.getString("url"),
 	                doc.getString("description"),
-	                doc.getString("postedBy"));
```
Comment (kaqqao) [inline content/backend/graphql-java/4-connectors.md]: Simply removing the line would make it invalid syntax. Should end up like: `doc.getString("description"));` instead.

**Comment by kaqqao** [comment]
Apart from the comment above, this is a good PR. Thanks!

**Comment by marktani** [comment]
Awesome, thanks @gadogttas and @kaqqao

**Pull request merged**